scripts/ci-integration.sh

```diff
@@ -33,6 +33,8 @@ CONTROLLER_REPO_KUBEADM_BOOTSTRAP="controller-ci-kubeadm-bootstrap"
 CONTROLLER_REPO_KUBEADM_CONTROL_PLANE="controller-ci-kubeadm-control-plane"
 EXAMPLE_PROVIDER_REPO="example-provider-ci"
 CERT_MANAGER_URL="https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml"
+TOOLS_DIR="hack/tools"
+TOOLS_BIN_DIR="${TOOLS_DIR}/bin"
```

Comment:
Are these definitions needed? They seem to replicate the defaults already defined in the Makefile.